apps/escrow/frontend/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite App</title>
```

Comment:
This title is automatically changed to FuelScan when the page is refreshed. Is this a bug?

Reply:
The title is overridden by the Layout component.  So I don't think we need to change anything here.

Reply:
This will need to be fixed later on